styleguide/components/21-atoms/logo/logo.twig

```diff
@@ -1 +1,6 @@
-<a href="#" title="Home" class="site-logo" rel="home"></a>
+<a href="#" title="Home" class="site-logo" rel="home">
+  {% include '@image' with {
+    'src': '/styleguide/img/svg/logo.svg',
+    'alt_text': 'Home'
```

Comment:
@delrueba is home correct?

Reply:
Yes, as long as it matches the desired navigation.